iOS-School-02_stack.playground/Contents.swift

```diff
@@ -0,0 +1,101 @@
+// MARK: - Stack
+/**
+ Provides a standard interface for operations on a collection,
+ which is organized according to the principle last in first out, called a `stack`.
+ Using class methods, it is possible to add an element to the collection,
+ remove an element from the collection and read the last element of the collection.
+ */
+class Stack {
+    /// An array with the elements of the stack collection.
+    private var stackElements = [Int]()
+    /**
+     Adds the element  to the end of the stack collection.
+
+     - Parameter element: An element to add on the end of the stack collection.
+     - Complexity: O(1)
+     */
+     func push(_ element: Int) {
```

Comment:
Indentation is wrong.
```suggestion
    func push(_ element: Int) {
```

Reply:
Fixed.